charts/tidb-backup/templates/backup-job.yaml

```diff
@@ -1,8 +1,8 @@
-{{- if .Values.backup.create }}
+{{- if eq .Values.mode "backup" }}
```

Comment:
Can you validate the mode? Maybe have `{{- if neq .Values.mode "restore" && neq .Values.mode "backup" }} ERROR`

Reply:
Yes, that should be added. But I can't find any document about `fail` or `error` in Helm template.

Reply:
I think there is a `fail` function. Helm says they have Sprig functions. http://masterminds.github.io/sprig/flow_control.html